src/track/annotation/AnnotationTrack.tsx

```diff
@@ -37,7 +37,8 @@ export class AnnotationTrack extends TrackObject<AnnotationTrackModel, Annotatio
     protected readonly macroLodThresholdLow = 7;
     protected readonly macroLodThresholdHigh = this.macroLodThresholdLow + this.macroLodBlendRange;
 
-    protected readonly namesLodBlendRange = 1;
+    //VALIS-8: increased 1->4 to show annotatation names at higher (zoomed out) LoD level
```

Comment:
"annotatation" should be "annotation"
I'd also add a space after //

Reply:
Fixed